spec/cocina/models/file_access_spec.rb

```diff
@@ -4,49 +4,51 @@
-                                        },
-                                        hasMessageDigests: []
-                                      }
-                                    ]
-                                  }
-                                }
-                              ]
-                            })
+    dro_template = Cocina::Models::DRO.new(externalIdentifier: 'druid:bc123df4567',
+                                           label: 'My DRO',
+                                           type: Cocina::Models::Vocab.book,
+                                           version: 1,
+                                           administrative: { hasAdminPolicy: 'druid:bc123df4567' },
+                                           access: {},
+                                           structural: {
+                                             contains: [
+                                               {
+                                                 version: 1,
+                                                 type: 'http://cocina.sul.stanford.edu/models/resources/file.jsonld',
+                                                 label: 'Page 1',
+                                                 externalIdentifier: 'abc123',
+                                                 structural: {
+                                                   contains: [
+                                                     {
+                                                       version: 1,
+                                                       type: 'http://cocina.sul.stanford.edu/models/file.jsonld',
+                                                       filename: '00002.jp2',
+                                                       label: '00002.jp2',
+                                                       hasMimeType: 'image/jp2',
+                                                       externalIdentifier: 'abc123',
+                                                       size: 111_467,
+                                                       administrative: {
+                                                         publish: true,
+                                                         sdrPreserve: true,
+                                                         shelve: true
+                                                       },
+                                                       access: {},
+                                                       hasMessageDigests: []
+                                                     }
+                                                   ]
+                                                 }
+                                               }
+                                             ]
+                                           })
+
+    dro_template.structural.contains.first.structural.contains.first.access = {
```

Comment:
The operation succeeds but the DRO fails to raise a validation error because a child structure was mutated and the child has no programmatic connection up to the parent to force a validation in the containing context, at least none that I can find.

Reply:
This may be related to file access validation being "relaxed," *i.e.* done by OpenAPI instead of by dry-struct? 

Worth a deeper look at a later date, perhaps. But perhaps not given the mutator interface reaches into the guts of dry-struct with relatively little payoff.

Cc: @justinlittman

Reply:
Both of these are true. Only some cocina models are validated on instantiation. And only some validation is performed by dry-struct (the OpenAPI validation is more expressive / powerful).